src/Unfactored/Roslyn/Core/RoslynDiagnosticsResources.resx

```diff
@@ -189,8 +189,8 @@
   <data name="CancellationTokenMustBeLastMessage" xml:space="preserve">
     <value>Method '{0}' should take CancellationToken as the last parameter</value>
   </data>
-  <data name="DoNotCallToImmutableArrayMessage" xml:space="preserve">
-    <value>Do not call ToImmutableArray on an ImmutableArray&lt;T&gt; value.</value>
+  <data name="DoNotCallToImmutableCollectionMessage" xml:space="preserve">
```

Comment:
Actually, on second though, no need to worry about the unfactored folder or anything in it.